packages/web/src/dev/LeaderBoard.tsx

```diff
@@ -97,10 +94,11 @@ function sorter(alpha: Competitor, bravo: Competitor) {
   return -1
 }
 
-const PORTIONS = 8
+const Portions = 8
```

Comment:
for consts all caps is really more common Pascal case I tent to think interfaces or classes

Reply:
true, the linter was complaining, but we can revert